<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SmartSettings</class>
 <widget class="QDialog" name="SmartSettings">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>551</width>
    <height>401</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>551</width>
    <height>401</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>551</width>
    <height>401</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Smartsettings</string>
  </property>
  <property name="windowIcon">
   <iconset resource="files.qrc">
    <normaloff>:/20x20/icons/20x20/cil-chart-line.png</normaloff>:/20x20/icons/20x20/cil-chart-line.png</iconset>
  </property>
  <widget class="QFrame" name="frame_main">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>551</width>
     <height>401</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>800</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame{
	background-color: rgb(0, 0, 0);
}

/* LINE EDIT */
QLineEdit {
	color: rgb(98, 103, 111);	
	background-color: rgb(27, 29, 35);
	border-radius: 5px;
	border: 2px solid rgb(27, 29, 35);
	padding-left: 10px;
}
QLineEdit:disabled {
	color: rgb(98, 103, 111);	
	background-color: rgb(27, 29, 35);
	border-radius: 5px;
	border: 2px solid rgb(27, 29, 35);
	padding-left: 10px;
}
QLineEdit:hover {
	border: 2px solid rgb(64, 71, 88);
}
QLineEdit:focus {
	border: 2px solid rgb(91, 101, 124);
}


/* SCROLL BARS */
QScrollBar:horizontal {
    border: none;
    background: rgb(52, 59, 72);
    height: 14px;
    margin: 0px 21px 0 21px;
	border-radius: 0px;
}
QScrollBar::handle:horizontal {
    background: rgb(85, 170, 255);
    min-width: 25px;
	border-radius: 7px
}
QScrollBar::add-line:horizontal {
    border: none;
    background: rgb(55, 63, 77);
    width: 20px;
	border-top-right-radius: 7px;
    border-bottom-right-radius: 7px;
    subcontrol-position: right;
    subcontrol-origin: margin;
}
QScrollBar::sub-line:horizontal {
    border: none;
    background: rgb(55, 63, 77);
    width: 20px;
	border-top-left-radius: 7px;
    border-bottom-left-radius: 7px;
    subcontrol-position: left;
    subcontrol-origin: margin;
}
QScrollBar::up-arrow:horizontal, QScrollBar::down-arrow:horizontal
{
     background: none;
}
QScrollBar::add-page:horizontal, QScrollBar::sub-page:horizontal
{
     background: none;
}
 QScrollBar:vertical {
	border: none;
    background: rgb(52, 59, 72);
    width: 14px;
    margin: 21px 0 21px 0;
	border-radius: 0px;
 }
 QScrollBar::handle:vertical {	
	background: rgb(85, 170, 255);
    min-height: 25px;
	border-radius: 7px
 }
 QScrollBar::add-line:vertical {
     border: none;
    background: rgb(55, 63, 77);
     height: 20px;
	border-bottom-left-radius: 7px;
    border-bottom-right-radius: 7px;
     subcontrol-position: bottom;
     subcontrol-origin: margin;
 }
 QScrollBar::sub-line:vertical {
	border: none;
    background: rgb(55, 63, 77);
     height: 20px;
	border-top-left-radius: 7px;
    border-top-right-radius: 7px;
     subcontrol-position: top;
     subcontrol-origin: margin;
 }
 QScrollBar::up-arrow:vertical, QScrollBar::down-arrow:vertical {
     background: none;
 }

 QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical {
     background: none;
 }


/* COMBOBOX */
QComboBox{
	color: rgb(210, 210, 210);
	background-color: rgb(27, 29, 35);
	border-radius: 5px;
	border: 2px solid rgb(27, 29, 35);
	padding: 5px;
	padding-left: 10px;
}
QComboBox:disabled{
	color: rgb(98, 103, 111);
	background-color: rgb(27, 29, 35);
	border-radius: 5px;
	border: 2px solid rgb(27, 29, 35);
	padding: 5px;
	padding-left: 10px;
}
QComboBox:hover{
	border: 2px solid rgb(64, 71, 88);
}
QComboBox::drop-down {
	subcontrol-origin: padding;
	subcontrol-position: top right;
	width: 25px; 
	border-left-width: 3px;
	border-left-color: rgba(39, 44, 54, 150);
	border-left-style: solid;
	border-top-right-radius: 3px;
	border-bottom-right-radius: 3px;	
	background-image: url(:/16x16/icons/16x16/cil-arrow-bottom.png);
	background-position: center;
	background-repeat: no-reperat;
 }
QComboBox QAbstractItemView {
	color: rgb(85, 170, 255);	
	background-color: rgb(27, 29, 35);
	padding: 10px;
	selection-background-color: rgb(39, 44, 54);
}

/* SLIDERS */
QSlider::groove:horizontal {
    border-radius: 9px;
    height: 18px;
	margin: 0px;
	background-color: rgb(52, 59, 72);
}
QSlider::groove:horizontal:hover {
	background-color: rgb(55, 62, 76);
}
QSlider::handle:horizontal {
    background-color: rgb(85, 170, 255);
    border: none;
    height: 18px;
    width: 18px;
    margin: 0px;
	border-radius: 9px;
}
QSlider::handle:horizontal:hover {
    background-color: rgb(105, 180, 255);
}
QSlider::handle:horizontal:pressed {
    background-color: rgb(65, 130, 195);
}

QSlider::groove:vertical {
    border-radius: 9px;
    width: 18px;
    margin: 0px;
	background-color: rgb(52, 59, 72);
}
QSlider::groove:vertical:hover {
	background-color: rgb(55, 62, 76);
}
QSlider::handle:vertical {
    background-color: rgb(85, 170, 255);
	border: none;
    height: 18px;
    width: 18px;
    margin: 0px;
	border-radius: 9px;
}
QSlider::handle:vertical:hover {
    background-color: rgb(105, 180, 255);
}
QSlider::handle:vertical:pressed {
    background-color: rgb(65, 130, 195);
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="pushButton_10">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>360</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>80</width>
      <height>30</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: 1px solid rgba(52, 59, 72, 150);
	border-radius: 5px;	
	background-color: rgba(52, 59, 72, 150);
}
QPushButton:hover {
	background-color: rgb(57, 65, 80);
	border: 1px solid rgb(61, 70, 86);
}
QPushButton:pressed {	
	background-color: rgb(35, 40, 49);
	border: 1px solid rgb(43, 50, 61);
}</string>
    </property>
    <property name="text">
     <string>Continue</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_11">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>360</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>80</width>
      <height>30</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: 1px solid rgba(52, 59, 72, 150);
	border-radius: 5px;	
	background-color: rgba(52, 59, 72, 150);
}
QPushButton:hover {
	background-color: rgb(57, 65, 80);
	border: 1px solid rgb(61, 70, 86);
}
QPushButton:pressed {	
	background-color: rgb(35, 40, 49);
	border: 1px solid rgb(43, 50, 61);
}</string>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_credits_9">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>19</y>
      <width>491</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(98, 103, 111);</string>
    </property>
    <property name="text">
     <string>As a result of audio analysis, the following audio parameters are determined.</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_credits_10">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>491</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(98, 103, 111);</string>
    </property>
    <property name="text">
     <string>To continue, click Continue or select the parameters manually.</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>200</y>
      <width>141</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_24">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>140</y>
      <width>341</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>250</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>350</width>
      <height>30</height>
     </size>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox{
	background-color: rgb(27, 29, 35);
	border-radius: 5px;
	border: 2px solid rgb(27, 29, 35);
	padding: 5px;
	padding-left: 10px;
}
QComboBox:hover{
	border: 2px solid rgb(64, 71, 88);
}
QComboBox QAbstractItemView {
	color: rgb(85, 170, 255);	
	background-color: rgb(27, 29, 35);
	padding: 10px;
	selection-background-color: rgb(39, 44, 54);
}</string>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
    <property name="currentIndex">
     <number>6</number>
    </property>
    <property name="maxCount">
     <number>20</number>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <property name="placeholderText" stdset="0">
     <string notr="true"/>
    </property>
    <item>
     <property name="text">
      <string>Telephone - 8 kHz - Lowest Quality</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Speech - 11,025 kHz - Lowest Quality</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Speech - 16 kHz - Low Quality</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Radio - 22,05 kHz - Low Quality</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Radio - 24 kHz - Low Quality</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>FM Radio - 32 kHz - Low Quality</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>CD Audio - 44.1 kHz - High Quality</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Digital audio tape - 48 kHz - High Quality</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>DVD Audio MLP 5.1 - 88.2 kHz - Studio Quality</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>DVD Audio MLP 5.1 - 96 kHz - Studio Quality</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>DVD Audio MLP 2.0 -176.4 kHz - Studio Quality</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>DVD Audio MLP 2.0 -192 kHz - Studio Quality</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_credits_13">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>140</y>
      <width>101</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(98, 103, 111);</string>
    </property>
    <property name="text">
     <string>Content type:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_25">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>200</y>
      <width>130</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>130</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>350</width>
      <height>30</height>
     </size>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox{
	background-color: rgb(27, 29, 35);
	border-radius: 5px;
	border: 2px solid rgb(27, 29, 35);
	padding: 5px;
	padding-left: 10px;
}
QComboBox:hover{
	border: 2px solid rgb(64, 71, 88);
}
QComboBox QAbstractItemView {
	color: rgb(85, 170, 255);	
	background-color: rgb(27, 29, 35);
	padding: 10px;
	selection-background-color: rgb(39, 44, 54);
}</string>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="maxCount">
     <number>20</number>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <property name="placeholderText" stdset="0">
     <string notr="true"/>
    </property>
    <item>
     <property name="text">
      <string>16 bit</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>24 bit</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_credits_15">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>200</y>
      <width>101</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(98, 103, 111);</string>
    </property>
    <property name="text">
     <string>Bit depth:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_credits_16">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>240</y>
      <width>101</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(98, 103, 111);</string>
    </property>
    <property name="text">
     <string>Compression:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_26">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>240</y>
      <width>130</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>130</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>350</width>
      <height>30</height>
     </size>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox{
	background-color: rgb(27, 29, 35);
	border-radius: 5px;
	border: 2px solid rgb(27, 29, 35);
	padding: 5px;
	padding-left: 10px;
}
QComboBox:hover{
	border: 2px solid rgb(64, 71, 88);
}
QComboBox QAbstractItemView {
	color: rgb(85, 170, 255);	
	background-color: rgb(27, 29, 35);
	padding: 10px;
	selection-background-color: rgb(39, 44, 54);
}</string>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
    <property name="currentIndex">
     <number>4</number>
    </property>
    <property name="maxCount">
     <number>20</number>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <property name="placeholderText" stdset="0">
     <string notr="true"/>
    </property>
    <item>
     <property name="text">
      <string>12</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>11</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>10</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>9</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>8 (Standart)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>7</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>6</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>0</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBox_27">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>100</y>
      <width>341</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>250</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>350</width>
      <height>30</height>
     </size>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox{
	background-color: rgb(27, 29, 35);
	border-radius: 5px;
	border: 2px solid rgb(27, 29, 35);
	padding: 5px;
	padding-left: 10px;
}
QComboBox:hover{
	border: 2px solid rgb(64, 71, 88);
}
QComboBox QAbstractItemView {
	color: rgb(85, 170, 255);	
	background-color: rgb(27, 29, 35);
	padding: 10px;
	selection-background-color: rgb(39, 44, 54);
}</string>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <property name="maxCount">
     <number>20</number>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <property name="placeholderText" stdset="0">
     <string notr="true">0</string>
    </property>
    <item>
     <property name="text">
      <string>Listening in the car on high-quality equipment</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Listening at home on standard equipment</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Listening at home on high-quality equipment</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Studio quality</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_credits_14">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>100</y>
      <width>101</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(98, 103, 111);</string>
    </property>
    <property name="text">
     <string>Optimize for:</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>300</y>
      <width>341</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Give preference to quality in doubtful cases</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="files.qrc"/>
 </resources>
 <connections/>
</ui>
